source/includes/_api_endpoint_contacts.md

```diff
@@ -407,6 +407,67 @@ Same as [Get Contact](#get-contact).
 > Note: In order to remove tag from contact add minus `-` before it.
 > For example: `tags: ['one', '-two']`  - sending this in request body will add tag `one` and remove tag `two` from contact.
 
+### Edit Batch Contact
+```php
+<?php
+
+$data = array(
+    array(
+        'id'        => 1,
+        'firstname' => 'Jim',
+        'lastname'  => 'Contact',
+        'email'     => 'jim@his-site.com',
+        'ipAddress' => $_SERVER['REMOTE_ADDR']
+    ),
+    array(
+        'id'        => 1,
+        'firstname' => 'John',
+        'lastname'  => 'Doe',
+        'email'     => 'john@his-site.com',
+        'ipAddress' => $_SERVER['REMOTE_ADDR']
+    )
+);
+
+$contact = $contactApi->editBatch($data);
+```
+Edit several contacts in one request.  Note that this supports PUT or PATCH depending on the desired behavior.
+
+** PUT ** creates a contact if the given ID does not exist and clears all the contact information, adds the information from the request.
```

Comment:
This might break Markdown formating
```suggestion
**PUT** creates a contact if the given ID does not exist and clears all the contact information, adds the information from the request.
```